easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

```diff
@@ -0,0 +1,63 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
+easyblock = 'ConfigureMake'
+version = '1.8_191-b26-OpenJDK'
+
+homepage = "https://openjdk.java.net/"
+description = """An open-source implementation of the Java Platform, Standard Edition"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/corba/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxp/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jaxws/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/langtools/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/jdk/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/nashorn/archive/',
+    'https://hg.openjdk.java.net/jdk8u/jdk8u/hotspot/archive/',
+]
+sources = [
+    '3322b7fdc03f.tar.gz',  # top-level repo
+    '314a4e05e420.tar.gz',  # corba
+    'a873d847260c.tar.gz',  # jaxp
+    'b0d7d6afebac.tar.gz',  # jaxws
+    '17bb956bc44e.tar.gz',  # langtools
+    '574e92cf225b.tar.gz',  # jdk
+    'ab9258f7206e.tar.gz',  # hotspot
+    'a53a027482b0.tar.gz',  # nashorn
+]
+checksums = [
+    '9ca2660a1000750829d4050a138ed4dd490a7a21919ed83fbde28d9b6a71bce4',  # 3322b7fdc03f.tar.gz
+    '961a83d51f1d302147f0178404ca5dd90a8944f5478cafaf5e0a4f970a7ae4ca',  # 314a4e05e420.tar.gz
+    '1209fc37a98985b9b315fceb17c0a2d6e986cedb15f364c7cd0ab4c3f51cd6d7',  # a873d847260c.tar.gz
+    '15649bb72d26be5f1ca137b2f421f49cde7d8b9df8a37887d0372cd4fef21012',  # b0d7d6afebac.tar.gz
+    '59b6c2bea249a9e75da91f9dd782f67a2478f1d6657e51d133f392622a04e432',  # 17bb956bc44e.tar.gz
+    'b8a7ee88e613e1e3785af5c398e258bad53e9cd03e88e39e1ee7f5701c40dd2f',  # 574e92cf225b.tar.gz
+    'd59b0235c2c18312c330a77eeea346642dbb18d8abcee405e60cd6a5d5812a86',  # ab9258f7206e.tar.gz
+    '5b897f75739bf14c702ae4cbf2b4afeeb160791ca0a6e06f4e1d5324db443429',  # a53a027482b0.tar.gz
+]
+
+osdependencies = ['java-1.8.0-openjdk-devel', 'cups-devel', 'alsa-lib-devel']
```

Comment:
Why is there a dependency for `java-1.8.0-openjdk-devel`, while we're installing exactly that?

Also, what's with the `cups` & `alsa` dependencies? Does `OpenJDK` really require printing & sound support to be in place in order for it to build & install?!